Beis.LearningPlatform.Web/Controllers/ComparisonToolController.cs

```diff
@@ -14,9 +14,10 @@ public class ComparisonToolController : ControllerBase
         }
 
         [Route("/comparison-tool")]
-        public async Task<IActionResult> Start()
+        [Route("/comparison-tool/{productCategoryIds}")]
```

Comment:
remove this line, make param optional

Reply:
productCategoryIds is already string parameter.

Also, if the main route is removed, then it would look for a cms page